gateway/ln-gateway/Cargo.toml

```diff
@@ -43,6 +43,7 @@ fedimint-core = { path = "../../fedimint-core" }
 fedimint-logging = { path = "../../fedimint-logging" }
 fedimint-rocksdb = { path = "../../fedimint-rocksdb" }
 futures = "0.3.24"
+lightning = "0.0.113"
```

Comment:
I would copy&paste this type, just to avoid another dependency.

Reply:
fixed in #2287